tests/docker-images/latest-version-image/Dockerfile

```diff
@@ -0,0 +1,52 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+FROM openjdk:8-jdk
+
+# Install some utilities
+RUN apt-get update && apt-get install -y netcat dnsutils supervisor python-kazoo
```

Comment:
one high-level question here:

unlike bookkeeper, pulsar builds latest docker image from the package. is it possible to let latest docker image depends on the latest package? 

I mean depending on the image here: https://github.com/apache/incubator-pulsar/tree/master/docker

If so, that means we are testing the image as well, right?

Reply:
Yes, the image can be built on the same build (it's different from the BK docker image). 

I agree with Sijie that we should try to reuse same image, either by deriving from that or, even, to just put the additional scripts in the final image if needed.

Reply:
If you're ok with that image being built via maven instead of scripts, that works for me.